modules/countries/api/countries-routes.ts

```diff
@@ -16,8 +16,38 @@ module.exports = function (app: core.Express) {
      *         schema:
      *           type: array
      *           items:
-     *             type: string
-     *             example: United Kingdom, United States Of America
+     *             type: object
```

Comment:
Now that we are returning an object, can you define a type for this and open api comment the type please.

Reply:
hi @ageddesi  i think it's a array of objects so this is fine imo